tests/unit/test_autoclaved.py

```diff
@@ -0,0 +1,62 @@
+import hashlib
+from measurements.pages import decompress_autoclaved
+
+# The s3 URL is something like:
+# https://s3.amazonaws.com/ooni-public/sanitised/2017-07-08/20170707T051609Z-IR-AS44244-web_connectivity-20170707T051545Z_AS44244_ujwFYcBJXcL2MjZnHXhBqVEOG2iHR0AuPUHB7aGGqUhdtvz70h-0.2.0-probe.json
+
+AUTOCLAVED = {
+    # This is the case of a report spanning many frames
+    # XXX this report is actually quite big maybe we want to pick as a test
+    # something smaller that still proves the point or maybe we write some code
+    # to generate it
+    'many_frames': {
+        'autoclaved_filename': '2017-07-08/20170707T051609Z-IR-AS44244-web_connectivity-20170707T051545Z_AS44244_ujwFYcBJXcL2MjZnHXhBqVEOG2iHR0AuPUHB7aGGqUhdtvz70h-0.2.0-probe.json.lz4',
+        'textname': '2017-07-08/20170707T051609Z-IR-AS44244-web_connectivity-20170707T051545Z_AS44244_ujwFYcBJXcL2MjZnHXhBqVEOG2iHR0AuPUHB7aGGqUhdtvz70h-0.2.0-probe.json',
+        'frame_off': 0,
+        'total_frame_size': 15485694 + 1724,
+        'intra_off': 0,
+        'report_size': 74695236,
+        # This doesn't actually match that which is in s3
```

Comment:
I do not understand the purpose of this comment? Also, below it's `autoclaved`, here `s3`, is that intentional?